bindata/etcd/pod.yaml

```diff
@@ -192,23 +192,12 @@ ${COMPUTED_ENV_VARS}
         memory: 600Mi
         cpu: 300m
     readinessProbe:
-      exec:
-        command:
-        - /bin/bash
-        - -c
-        - |
-          set -xe
-
-          # Unix sockets are used for health checks to ensure that the pod is reporting readiness of the etcd process
-          # in this container. While this might seem unnecessary the use of SO_REUSEADDR has made this explicitly
-          # required as the kernel will allow the reuse of a port while in TIME_WAIT. etcd requires socket
-          # path in this format <name>:<port> so port 0 is used only to meet this requirement.
-          unset ETCDCTL_ENDPOINTS
-          /usr/bin/etcdctl \
-            --command-timeout=2s \
-            --dial-timeout=2s \
-            --endpoints=unixs://${NODE_NODE_ENVVAR_NAME_IP}:0 \
-            endpoint health -w json | grep \"health\":true
+      httpGet:
+        port: 9980
+        path: readyz
```

Comment:
are we going to hit this endpoint from the operator in the future?

Reply:
Possibly but this is mostly for the etcd pod's own readiness probe for the `etcd` container and also for the guard pod to check readiness.
https://github.com/openshift/library-go/blob/master/pkg/operator/staticpod/controller/guard/manifests/guard-pod.yaml#L42-L46

Although the guard pod is using the outdated `/healthz` path.